### PR TITLE
Provide more detail on class projects

### LINKED ISSUE #511: Add materials to 'Project Proposals' week

### PR DESCRIPTION
**Pull request by ethanwhite**
Close #511

### CODE REVIEW

File: materials/projects.md [hunk 1]
```diff
@@ -5,13 +5,40 @@ title: Projects - Intro
...
-* Can be on anything
-* Should be something you are excited about
-* 30-40 hours of work
+## Goals
 
-* Email me a <1 page proposal whenever you're ready or set up some time to chat
-* No later than November 1st
+Course projects are intended to give you the opportunity to engage in a larger
+project on a topic of your choosing. If you are a graduate student or an
+undergraduate researcher choosing a project that is directly related to your
```
Comment (brymz) [inline materials/projects.md]: "If you are a graduate student or an undergraduate researcher, we encourage you to choose a project that is directly related to your research."

Comment (ethanwhite) [inline materials/projects.md]: Fixed.

**Comment by ethanwhite** [comment]
Where to put this is a good question. I only had a full week for this this year because I happened to be traveling. It could just go in the syllabus or we could just have a direct link to it under the `For Students` section of the home page. Thoughts?

**Comment by brymz** [comment]
It would not be a trouble to do both. Should I remove the "Project Proposal" week from the schedule then?

**Comment by ethanwhite** [comment]
:+1:

**Pull request merged**